pkg/apis/kubermatic/v1/datacenter.go

```diff
@@ -412,22 +412,22 @@ type NodeSettings struct {
 	ProxySettings `json:",inline"`
 	// Optional: These image registries will be configured as insecure
 	// on the container runtime.
-	InsecureRegistries []string `json:"insecure_registries,omitempty"`
+	InsecureRegistries []string `json:"insecureRegistries,omitempty"`
 	// Optional: These image registries will be configured as registry mirrors
 	// on the container runtime.
-	RegistryMirrors []string `json:"registry_mirrors,omitempty"`
+	RegistryMirrors []string `json:"registryMirrors,omitempty"`
 	// Optional: Translates to --pod-infra-container-image on the kubelet.
 	// If not set, the kubelet will default it.
-	PauseImage string `json:"pause_image,omitempty"`
+	PauseImage string `json:"pauseImage,omitempty"`
 	// Optional: The hyperkube image to use. Currently only Flatcar
 	// makes use of this option.
-	HyperkubeImage string `json:"hyperkube_image,omitempty"`
+	HyperkubeImage string `json:"hyperkubeImage,omitempty"`
 }
 
 // SeedMLASettings allow configuring seed level MLA (Monitoring, Logging & Alerting) stack settings.
 type SeedMLASettings struct {
 	// Optional: UserClusterMLAEnabled controls whether the user cluster MLA (Monitoring, Logging & Alerting) stack is enabled in the seed.
-	UserClusterMLAEnabled bool `json:"user_cluster_mla_enabled,omitempty"`
+	UserClusterMLAEnabled bool `json:"userClusterMlaEnabled,omitempty"`
```

Comment:
As discussed in our meeting, let's turn this into an extendable struct, like

```yaml
seedMLASettings:
  userCluster: # also no need to call it "userClusterMLA" when it's already a child of `seedMLASettings` i guess
    enabled: true
```

Reply:
This PR is only about renaming tags and group.

P.S.
The tag seems to be renamed incorrectly, shoulda be `userClusterMLAEnabled`.

Reply:
And I don't think we can do API structure changes like this, before we start using this new API.